lib/php-version.js

```diff
@@ -173,3 +173,46 @@ function numberedVersionData(version) {
     modifierCount: modifierLevelCount,
   };
 }
+
+function isHhvm(version) {
+  return version.indexOf('hhvm') === 0;
```

Comment:
Happily, we now have [`String#startsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith). So I'm not sure you even need a helper function for this anymore.

Instead of:
```js
const hasHhvm = travisVersions.find((v) => phpHhvmVersion(v));
```
You can write:
```js
const hasHhvm = travisVersions.find((v) => v.startsWith('hhvm'));
```
which is arguably more readable.